warehouse/query-core/src/main/java/datawave/query/tables/RangeStreamScanner.java

```diff
@@ -61,7 +75,7 @@
     // simply compare the strings. no need for a date formatter
     protected static final int dateCfLength = 8;
     protected boolean seenUnexpectedKey = false;
-    protected Queue<Entry<Key,Value>> currentQueue;
+    protected ArrayDeque<Entry<Key,Value>> currentQueue;
```

Comment:
these are not thread safe

Reply:
They never have been -- from what I can tell this is 'acceptable'.